### ISSUE TITLE
[Enhancement] Automatically update .classpath file

### ISSUE THREAD

**Issue by FunshineX** [Microsoft/vsminecraft#22]
When I rename packages, add subfolders, etc, the .classpath file is not updated.  I have to manually add additional classpathentry tags so the intellisense works and the project compiles.

**Comment by mluparu** [comment]
Yes, only the other direction (from classpath to VS project) is currently implemented (and even that is through the manual commands described in #7). 

As a workaround: As long as you add the packages and additional source folders in the gradle.build file rather than the VS project, after running a build and the CustomCommands.ProvisionMinecraftProject, you shoudl have both an updated .classpath (that Intellisense can work with) as well as the VS project (that debugger needs to launch correctly).
